### ISSUE TITLE
Messy output 

### ISSUE THREAD

**Issue by SmileB4Deathz** [tsionyx/nonogrid#2]
Running the example  `cargo run --features=web,xml -- --webpbn 5933` 
returns the following output: https://imgur.com/a/aZk6CVs

or `cargo run --features=web,xml -- --webpbn 5933` shows the problem better:
https://imgur.com/a/XbpzR40

I've tried both cmder and the default windows console.

**Comment by tsionyx** [comment]
The first image gets messed up because your terminal window is not wide enough.

As for the second, things are more complicated: all I know is that is the issue with the gnome-terminal. For some reason, it shows the Unicode character ⬛ (large filled square) with width=2 characters while rust considers it has the size of 1 character.

I checked my terminal and it shows the same picture as yours now, but 2 years ago, when I took [this screenshot for the README](https://github.com/tsionyx/nonogrid/blob/master/examples/rust_logo.png), it was all fine.
